### PR TITLE
Add blog post 'Anatomy of a Serverless Application'

### PR DESCRIPTION
**Pull request by rupakg**

### CODE REVIEW

File: posts/2017-07-24-anatomy-of-a-serverless-app.md [hunk 1]
```diff
@@ -0,0 +1,669 @@
...
+{"message":"Bad input data or missing email address.","input":"{\"junk\"}"}
+```
+
+**No Email Address**
+
+```
+$ curl -i -X POST -d '{"to_address":""}' https://yt1i5ydiu4.execute-api.us-east-1.amazonaws.com/prod/email
+
+{"message":"Bad input data or missing email address.","input":"{\"to_address\":\"\"}"}
+```
```
Comment (alexdebrie) [inline posts/2017-07-24-anatomy-of-a-serverless-app.md]: These four examples are pretty similar -- maybe condense to 2?

File: posts/2017-07-24-anatomy-of-a-serverless-app.md [hunk 2]
```diff
@@ -0,0 +1,669 @@
...
+START RequestId: 0eb38aeb-6d9d-11e7-9787-2996c7e3fdb0 Version: $LATEST
+2017-07-20 22:44:58.544 (+00:00)	0eb38aeb-6d9d-11e7-9787-2996c7e3fdb0	{ Error: 'to' parameter is not a valid address. please check documentation
+...
+...
+2017-07-20 22:44:58.545 (+00:00)	0eb38aeb-6d9d-11e7-9787-2996c7e3fdb0	{"errorMessage":"'to' parameter is not a valid address. please check documentation","errorType":"Error","stackTrace":
+...
+...
+END RequestId: 0eb38aeb-6d9d-11e7-9787-2996c7e3fdb0
+REPORT RequestId: 0eb38aeb-6d9d-11e7-9787-2996c7e3fdb0	Duration: 339.84 ms	Billed Duration: 400 ms 	Memory Size: 1024 MB	Max Memory Used: 41 MB
+
```
Comment (alexdebrie) [inline posts/2017-07-24-anatomy-of-a-serverless-app.md]: It's nice to show how unhandled errors manifest themselves w/ Lambda, but don't know that we want to encourage this. Maybe add a section to refactor and catch that error and return something meaningful to the user?

File: posts/2017-07-24-anatomy-of-a-serverless-app.md [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+  |-- README.md
+  |-- frontend
+  `-- services
+```
+where,
+
+  * `frontend` folder holds the frontend application
+  * `services` folder holds the serverless service(s)
+
+A structure like this provides clear separation for the non-serverless and serverless code for the overall application. The way the non-serverless portion of the application is written is totally your choice. In this post, we will primarily focus on the serverless portion of the application inside the `services` folder.
```
Comment (alexdebrie) [inline posts/2017-07-24-anatomy-of-a-serverless-app.md]: The `frontend` part is confusing to me since it's not referenced in the post at all.

**Comment by rupakg** [comment]
@alexdebrie Thanks for your feedback. I have updated the post based on your comments, fixed typos, and refactored code to add exception handling at the end.

**Comment by rupakg** [comment]
@andrea-p Thanks for the review. Nice edits!

**Comment by andrea-p** [comment]
schedule(07/27/2017 6:30 am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **07/27/2017 at 6:30am (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/223#issuecomment-318181624)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**